corehq/apps/locations/templates/locations/partials/downgrade_locations.html

```diff
@@ -0,0 +1,10 @@
+{% load i18n %}
+{% load hq_shared_tags %}
+
+<h1>Locations have been downgraded.</h1>
+
+<form action="{% url "unassign_users" request.domain %}" method="POST">
+   <input type="hidden" name="redirect" value="{{ request.path }}">
+   <input value="{% trans "Unassign all users from locations" %}"
+          type="submit" class="btn btn-danger"></input>
+</form>
```

Comment:
This file is implemented as a partial so it can be dropped in to a few different places.  The actual display text needs to be updated by @amsagoff or @sheelio .